### PR TITLE
Uplift to k8s 1.21.1

### PR DESCRIPTION
**Pull request by Xenwar**
Uplift to k8s 1.21.1

### CODE REVIEW

File: vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc [hunk 1]
```diff
@@ -46,6 +46,24 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
       - systemctl enable --now keepalived
       - chown {{ IMAGE_USERNAME }}:{{ IMAGE_USERNAME }} /home/{{ IMAGE_USERNAME }}/.kube/config
     files:
+      - path: /usr/local/bin/retrieve.configuration.files.sh
```
Comment (kashifest) [inline vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc:49]: We don't need this here. Its already baked in the image

Comment (smoshiur1237) [reply vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc:49]: We find the issue in node image script in kubernetes version.

Comment (Xenwar) [reply vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_ubuntu.rc]: Removed it and pushed a new version

**Comment by kashifest** [comment]
This might cause failure due to kubeadm missing the etcd patch as was used [here](https://artifactory.nordix.org/artifactory/airship/kubeadm_etcd_patched/k8s_v1.21.0/kubeadm). This will also cause the upgrade and feature test to fail. So which one shall we take in first to cause less red builds in CI? This or #666.

**Comment by furkatgofurov7** [comment]
> This might cause failure due to kubeadm missing the etcd patch as was used [here](https://artifactory.nordix.org/artifactory/airship/kubeadm_etcd_patched/k8s_v1.21.0/kubeadm). This will also cause the upgrade and feature test to fail. So which one shall we take in first to cause less red builds in CI? This or #666.

I would say #666 must go in first.

**Comment by Xenwar** [comment]
> > This might cause failure due to kubeadm missing the etcd patch as was used [here](https://artifactory.nordix.org/artifactory/airship/kubeadm_etcd_patched/k8s_v1.21.0/kubeadm). This will also cause the upgrade and feature test to fail. So which one shall we take in first to cause less red builds in CI? This or #666.
> 
> I would say #666 must go in first.

Ok with #666 going in first. 
I have already included this commit in it. However, all tests are not passing there.

File: lib/images.sh [hunk 2]
```diff
@@ -3,17 +3,17 @@
 # Image name and location
 IMAGE_OS=${IMAGE_OS:-Centos}
 if [[ "${IMAGE_OS}" == "Ubuntu" ]]; then
-  export IMAGE_NAME=${IMAGE_NAME:-UBUNTU_20.04_NODE_IMAGE_K8S_v1.21.0.qcow2}
-  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://artifactory.nordix.org/artifactory/airship/images/k8s_v1.21.0/}
+  export IMAGE_NAME=${IMAGE_NAME:-UBUNTU_20.04_NODE_IMAGE_K8S_v1.21.1.qcow2}
+  export IMAGE_LOCATION=${IMAGE_LOCATION:-https://artifactory.nordix.org/artifactory/airship/images/k8s_v1.21.1/}
```
Comment (smoshiur1237) [inline lib/images.sh:7]: If the parameterize PR get merged first,  the file doesn't need any changes in this PR.

**Comment by smoshiur1237** [comment]
At this point, looks good to me. Lets wait for the CI run.

File: scripts/feature_tests/inspection_test.sh [hunk 3]
```diff
@@ -4,6 +4,11 @@ set -x
 
 METAL3_DIR="$(dirname "$(readlink -f "${0}")")/../.."
 
+# shellcheck disable=SC1091
+# shellcheck disable=SC1090
+# shellcheck disable=SC2046
+source "feature_test_vars.sh"
```
Comment (kashifest) [inline scripts/feature_tests/inspection_test.sh:10]: This is still not working

Comment (kashifest) [reply scripts/feature_tests/inspection_test.sh:10]: Since we have `METAL3_DIR` can we source it with respect to that in all the files

Comment (Xenwar) [reply scripts/feature_tests/inspection_test.sh]: Noted. Thanks

Comment (smoshiur1237) [reply scripts/feature_tests/inspection_test.sh]: Good catch.

**Review by smoshiur1237 — APPROVED** [review]
/lgtm

**Pull request merged**